scraper/src/typesense_helper.py

```diff
@@ -93,28 +180,31 @@ def add_records(self, records, url, from_sitemap):
 
         print(
             '\033[{}m> DocSearch: \033[0m{}\033[93m {} records\033[0m)'.format(
-                color, url, record_count))
+                color, url, record_count
+            )
+        )
 
     def commit_tmp_collection(self):
         """Update alias to point to new collection"""
-        old_collection_name = None
+        old_collection_name = self._get_old_collection_name()
 
-        try:
-            old_collection_name = self.typesense_client.aliases[self.alias_name].retrieve()['collection_name']
-        except exceptions.ObjectNotFound:
-            pass
-
-        self.typesense_client.aliases.upsert(self.alias_name, {'collection_name': self.collection_name_tmp})
+        self.typesense_client.aliases.upsert(
```

Comment:
@tharropoulos Could we only update the alias and point it to the new collection, once all the synonyms and curation rules have been transferred over? 

Because as it currently stands, there will be a brief moment when searches will be sent to the new collection, without synonyms and curation rules copied over yet.

Reply:
This creates side effects indeed, resolved in 41aa511fb5b70fcf90f9ffd0f0667e2996e34827. Still kept the deletion of the old collection being executed last in the workflow.